src/protocol/definitions/ping360.json

```diff
@@ -28,34 +28,36 @@
                         "description": "Operating mode (1 for Ping360)"
                     },
                     {
-                        "name": "gain_index",
+                        "name": "gain_setting",
                         "type": "u8",
-                        "description": "Analog gain index (0 = low, 1 = normal, 2 = high)"
+                        "description": "Analog gain setting (0 = low, 1 = normal, 2 = high)"
                     },
                     {
                         "name": "angle",
                         "type": "u16",
-                        "description": "Head angle in gradians (3 ASCII/Hex digits 0 to 399 decimal = 0 to 360 degrees)"
+                        "description": "Head angle",
+                        "units": "gradian"
                     },
                     {
-                        "name": "pulse_usec",
+                        "name": "transmit_duration",
```

Comment:
https://github.com/bluerobotics/ping-protocol/pull/151#discussion_r288526358

Reply:
this will break ping-viewer and can be done subsequently (independently). Do you want to hold the pr for this?

Reply:
It'll not necessary "break" ping-viewer since we works with submodules, besides that we are also working with device-manager branch to test and work with ping360.

I'm more worried about ping-arduino and ping-protocol than Ping-Viewer. If you think if that's ok, we should update ping1D messages.